Makefile

```diff
@@ -185,10 +189,14 @@ storage:
 
 .PHONY: kafka
 kafka:
+ifeq ($(OLM),true)
+	@echo Skipping Kafka deployment, assuming it has been installed via OperatorHub
```

Comment:
nit: change "Kafka deployment" to "kafka-operator deployment"

Reply:
Updated.  I also realized that we shouldn't do undeploy-kafka or undeploy-es-operator when testing with OperatorHub so I have updated those steps too.